### PR TITLE
Address missing PR comments for Add new telemetries to track events related to PMC and Powershell usage

### PR DESCRIPTION
**Pull request by erdembayar**
## Bug
Fixes: NuGet/Home[Issue#10478](https://github.com/NuGet/Home/issues/10478)
Regression: No
* Last working version:   
* How are we preventing it in future:   

## Fix

Details: 
For [issue#10142](https://github.com/NuGet/Home/issues/10142) I received PR review from @nkolev92 right after I merged my [PR#3755](https://github.com/NuGet/NuGet.Client/pull/3755). 
There are 3 comments left by @nkolev92 need to be addressed.
- [x] https://github.com/NuGet/NuGet.Client/pull/3755#discussion_r562995245 Convert dictionary to class
- [x] https://github.com/NuGet/NuGet.Client/pull/3755#discussion_r562995592 Convert dictionary to class
- [x] https://github.com/NuGet/NuGet.Client/pull/3755#discussion_r563939547 Use static import

Testing/Validation
Tests Added: No
Reason for not adding tests: Not sure how to trigger VS to load PMC window, PMUI window before and after solution open.
Validation: Manually tested with "VS Telemetry Monitor"

### CODE REVIEW

**Comment by erdembayar** [comment]
@nkolev92 
Please review this one. I addressed other 2 comments on previous [PR](https://github.com/NuGet/NuGet.Client/pull/3755).

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
+{
+    public class InstanceCloseEvent : TelemetryEvent
+    {
+        public InstanceCloseEvent(
+            int PmcExecuteCommandCount,
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: Parameter names are usually `camelCase`. We don't seem to have an explicit rule in https://github.com/NuGet/NuGet.Client/blob/dev/docs/coding-guidelines.md, but we I will add it.

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: More than 1 file has this issue, please address them all.

**Review by nkolev92 — COMMENTED** [review]
Thanks for creating a new issue + PR. 

Please follow the PR template and keep the checklist.

https://raw.githubusercontent.com/NuGet/NuGet.Client/dev/.github/PULL_REQUEST_TEMPLATE.md

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs (hunk 1 above)
Comment (erdembayar) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: @nkolev92 
Addressed the issue. 
I was receiving comments on properties about Pascal casing then I used replace all to address it, then this is byproduct of it.

**Review by erdembayar — COMMENTED** [review]
@nkolev92 
Thank you for your review.
Addressed you latest review comments.

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs [hunk 2]
```diff
@@ -0,0 +1,29 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: The namespace doesn't match the assembly

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/PowerShellLoadedEvent.cs [hunk 3]
```diff
@@ -0,0 +1,16 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/PowerShellLoadedEvent.cs]: namespace doesn't match the assembly.

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/SolutionCloseEvent.cs [hunk 4]
```diff
@@ -0,0 +1,38 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using NuGet.Common;
+
+namespace NuGet.VisualStudio.Telemetry.PowerShell
```
Comment (nkolev92) [inline src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/SolutionCloseEvent.cs]: namespace doesn't match the assembly.

**Review by nkolev92 — APPROVED** [review]
Some more coding style feedback before merging.

Namespace didn't match the assembly.

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs (hunk 2 above)
Comment (erdembayar) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: @nkolev92 

Ok. Corrected. Previously I checked nearby other files in same project and took it.

https://github.com/NuGet/NuGet.Client/blob/59792e4733ced999af222a8ba7adb7a4ee8f5a08/src/NuGet.Clients/NuGet.VisualStudio.Common/UserInterfaceService/INuGetUILogger.cs#L6

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: The namespace should be the assembly name. 

The INuGetUILogger type was likely moved recently from some other location *to* the NuGet.VisualStudio.Common.

Comment (nkolev92) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/InstanceCloseEvent.cs]: https://github.com/NuGet/NuGet.Client/blob/dev/src/NuGet.Clients/NuGet.VisualStudio.Common/INuGetErrorList.cs

I will add that to the guidelines as well.

File: src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/PowerShellLoadedEvent.cs (hunk 3 above)
Comment (nkolev92) [reply src/NuGet.Clients/NuGet.VisualStudio.Common/Telemetry/Powershell/PowerShellLoadedEvent.cs]: https://github.com/NuGet/NuGet.Client/pull/3867

**Pull request merged**